### DISCUSSION TITLE
default snmp.yml results in parsing config file errors

### DISCUSSION THREAD

**Discussion by bcheevers123** [prometheus/snmp_exporter#875] [Q&A] [answered]
Hello, when I run

prometheus-snmp-exporter --config.file=/home/pi/Desktop/snmpExport/snmp.yml 

I get FATAL Error parsing config file: yaml: unmarshal error:
`  line 50: field enum_values not found in type config.Metric
`

I am using the default snmp.yml file in the repo.

**Comment by SuperQ** [comment, marked as answer]
This is likely because you are using the Debian package, which contains a very old version of the exporter which doesn't support modern features.
